tests/attested_tls/sgxlkl_enclave/makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright Microsoft. 
+# Licensed under the attached Microsoft Software License Terms
+
+.PHONY: all clean alpine-container alpine-binaries lkl-image lkl-run
+
+COMMONS="../common"
+SGXLKL_BIN_ROOT="$(SGXLKL_PREFIX)/bin"
+BUILD_DIR="/src/sgxlkl_enclave"
+SERVER_COMMON="/src/common"
+
+build-alpine-exe = sudo gcc -g -o $(BUILD_DIR)/alpine_server -fshort-wchar $(BUILD_DIR)/client.c $(BUILD_DIR)/tlscli.c $(SERVER_COMMON)/settings.c -lmbedtls -lmbedx509 -lmbedcrypto -ldl -lcurl
+
+all: lkl-image
+
+clean:
+	rm -f clientAgent.o client alpine_server sgxlkl-image.img*
+
+alpine-container:
+	docker build -t alpine3.10.build ./alpine
+
+alpine-binaries: alpine-container
```

Comment:
Follow #664, try to remove  `alpine-container`, `alpine-binaries`,  the alpine folder.